catalyst/engines/apex.py

```diff
@@ -485,14 +439,9 @@ def init_components(
         optimizer = optimizer_fn()
         optimizer = self.sync_device(optimizer)
 
-        model, optimizer = amp.initialize(
-            model,
-            optimizer,
-            opt_level=self.opt_level,
-            keep_batchnorm_fp32=self.keep_batchnorm_fp32,
-            loss_scale=self.loss_scale,
-        )
-        model = ApexDistributedDataParallel(model, delay_allreduce=self.delay_all_reduce)
+        model, optimizer = amp.initialize(model, optimizer, **self.apex_kwargs)
+        # TODO: kwargs for Apex DDP ?
+        model = ApexDistributedDataParallel(model)  # , delay_allreduce=self.delay_all_reduce)
```

Comment:
could we also add `ddp_kwargs` and pass them here?

Reply:
in this case, we have to remove ** from the __init__ and make `apex_krargs` and `ddp_kwargs` dict storages